{
	"entries": [
		{
				"message": "Running Connect.script",
				"identifier": "START",
				"timestamp": "2023-09-11 14:08:59.416 +0800"
			},
		{
				"message": "(Duration:\"0.288\", Errors:\"1\", Exceptions:\"0\", StartTime:\"2023-09-11 14:08:59 +0800\", Successes:\"0\", TestCase:\"Connect.script\", Warnings:\"0\")",
				"identifier": "EndTestCase",
				"timestamp": "2023-09-11 14:08:59.721 +0800"
			},
		{
				"message": "STMaxCallDepthExceeded: Handlers were called to a depth exceeding the maximum limit (40). HINT: One common cause for this error is a script that has been given the same name as a built-in command or function unintentionally calling itself. -- Execution Time 0:00:00 Connect.script",
				"identifier": "FAILURE",
				"timestamp": "2023-09-11 14:08:59.754 +0800"
			}
	],
	"createdTimestamp": "2023-09-11 14:08:59.420 +0800"
}